release/src/main/scripts/build_release_candidate.sh

```diff
@@ -106,9 +106,7 @@ if [[ $confirmation = "y" ]]; then
                 -Prelease.releaseVersion=${RELEASE}-RC${RC_NUM} \
                 -Prelease.useAutomaticVersion=true --info --no-daemon
 
-  echo "Please make sure gradle release succeed: "
-  echo "1. release code has been pushed to github repo."
-  echo "2. new rc tag has created in github."
+  git push "v${RELEASE}-RC${RC_NUM}"
```

Comment:
When I did this locally there was one more commit on the `release-2.22.0` branch changing the version to 2.22.0-SNAPSHOT, so I had to run `git push origin release-2.22.0` as well.

Also is it ok to omit the `origin`?

Reply:
> When I did this locally there was one more commit on the release-2.22.0 branch changing the version to 2.22.0-SNAPSHOT, so I had to run git push origin release-2.22.0 as well.

I see two commits, one changing it to RC1 and next changing it back to SNAPSHOT. Isn't that just a no-op? https://github.com/apache/beam/commits/release-2.22.0

> Also is it ok to omit the origin?

Origin is the default IIUC.

Reply:
> I see two commits, one changing it to RC1 and next changing it back to SNAPSHOT. Isn't that just a no-op? https://github.com/apache/beam/commits/release-2.22.0

Yes but the first one is tagged with v2.22.0-RC1 https://github.com/apache/beam/commits/v2.22.0-RC1. So if you only push the tag it won't actually be a noop.

> Origin is the default IIUC.

`git help push` says `git push ... [<repository> [<refspec>...]]` which I take to mean if you only pass one arg it's interpreted as the `repository`. That does seem to be what happens when I try it:
```
❯ git push release-2.22.0
fatal: 'release-2.22.0' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Reply:
Ah you're right about both. Fixed, PTAL